backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -267,19 +267,14 @@ fieldset {
       }
     }
 
-    span.or {
-      background-color: $color-1;
-      border-width: 5px;
-      margin-left: 5px;
-      margin-right: 11px;
-      position: relative;
-      z-index: 1;
-
-      -webkit-box-shadow: 0 0 0 5px $color-1;
-         -moz-box-shadow: 0 0 0 5px $color-1;
-          -ms-box-shadow: 0 0 0 5px $color-1;
-           -o-box-shadow: 0 0 0 5px $color-1;
-              box-shadow: 0 0 0 5px $color-1;
+    // Previously there had been a span of class 'or' used for spacing.
```

Comment:
So close! This makes a good commit message but it's entirely what's needed here. The goal here is to explain what this code is doing so that the next person looking at this doesn't have to scratch their head for 10min. Something like:

```scss
// Always make sure buttons are spaced out. Sometimes there are forms
// with single buttons inside so we can't always just do `.button + .button`.
```